commit c6430386f19ac72ba128682bebbb542d58ceb902
Author: Trent Mick <trentm@gmail.com>
Date:   2019-07-30T09:10:48-07:00 (10 weeks ago)
    
    MANTA-4439 'manta-shardadm list' should include set BUCKETS_MORAY_SHARDS
    Reviewed by: Isaac Davis <isaac.davis@joyent.com>
    Approved by: Isaac Davis <isaac.davis@joyent.com>

diff --git a/cmd/manta-shardadm.js b/cmd/manta-shardadm.js
index 52b77d4..15dabb9 100755
--- a/cmd/manta-shardadm.js
+++ b/cmd/manta-shardadm.js
@@ -100,12 +100,13 @@ Shardadm.prototype.do_list.help = (
 );
 
 function printShards(metadata, cb) {
+	var i;
 	var fmt = '%-12s %s';
 
 	console.log(sprintf(fmt, 'TYPE', 'SHARD NAME'));
 
 	if (metadata[common.INDEX_SHARDS]) {
-		for (var i = 0; i < metadata[common.INDEX_SHARDS].length; i++) {
+		for (i = 0; i < metadata[common.INDEX_SHARDS].length; i++) {
 			console.log(sprintf(fmt,
 			    'Index', metadata[common.INDEX_SHARDS][i].host));
 		}
@@ -118,6 +119,13 @@ function printShards(metadata, cb) {
 		console.log(sprintf(fmt,
 		    'Storage', metadata[common.STORAGE_SHARD]));
 
+	if (metadata[common.BUCKETS_SHARDS]) {
+		for (i = 0; i < metadata[common.BUCKETS_SHARDS].length; i++) {
+			console.log(sprintf(fmt, 'Buckets',
+			    metadata[common.BUCKETS_SHARDS][i].host));
+		}
+	}
+
 	return (cb(null));
 }
 
diff --git a/package.json b/package.json
index 9de8c88..53fee8d 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "manta-deployment",
     "description": "Manta deployment configuration",
-    "version": "0.0.6",
+    "version": "0.0.7",
     "author": "Joyent (joyent.com)",
     "license": "MPL-2.0",
     "private": true,
